src/test/java/io/logz/LogzioSenderTest.java

```diff
@@ -67,11 +67,11 @@ public void simpleAppending() throws Exception {
         // Sleep double time the drain timeout
         Thread.sleep(drainTimeout * 1000 * 2);
 
-        assertTrue(mockListener.checkForLogExistence(token, type, loggerName, Level.INFO, message1, false, null));
-        assertTrue(mockListener.checkForLogExistence(token, type, loggerName, Level.WARN, message2, false, null));
+        assertTrue(mockListener.checkForLogExistence(token, type, loggerName, Level.INFO, message1, false, null, null));
```

Comment:
Just add mockListener.checkForLogExistence(...) which calls the original with null (or Optional.empty()) thus you don't need to change your code.